stable/superset/values.yaml

```diff
@@ -63,6 +63,10 @@ extraEnv: {}
 ##
 extraEnvFromSecret: ""
 
+## Deployment level annotations
+## Useful for passing other third party annotations to interact with eg. kube2iam.
+annotations: {}
```

Comment:
we should probably name this variable `deploymentAnnotations` or similar to ensure folks know its for the deployment and not service or ingress or otherwise.

Reply:
thanks, I have made the change. Could you resolve if you are satisfied?